<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDiseasesWindow</class>
 <widget class="QDialog" name="AddDiseasesWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>444</width>
    <height>478</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Epidemics Data Entry Form</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>31</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Epidemics Data Entry Form</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>291</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Date Of Entry</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>261</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Region</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>261</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Disease</string>
      </property>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>261</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Number Of Cases</string>
      </property>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>131</width>
         <height>25</height>
        </rect>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>420</y>
     <width>178</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddDiseasesWindow</receiver>
   <slot>addCaseSubmitButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AddDiseasesWindow</receiver>
   <slot>addCaseCancelButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addCaseSubmitButtonHandler()</slot>
  <slot>addCaseCancelButtonHandler()</slot>
 </slots>
</ui>
